pkg/network/config/config.go

```diff
@@ -0,0 +1,131 @@
+package config
+
+import (
+	"time"
+
+	"github.com/DataDog/datadog-agent/pkg/ebpf"
+)
+
+// Config stores all flags used by the network eBPF tracer
+type Config struct {
```

Comment:
Really happy to see this moved to a separate package. There are a bunch of constructors that can be simplified now (eg. `NewSocketFilterSnooper`)